ProjectHeads/GenerateSingleSampleHeads.ps1

```diff
@@ -1,30 +1,48 @@
 <#
 .SYNOPSIS
     Uses the dotnet template tool to copy and rename project heads to run sample code for different platforms.
+
 .DESCRIPTION
     This is used to centralize configuration and reduce duplication of copying these heads for every project.
-
     This script also generates a solution for the project and will open Visual Studio.
+
 .PARAMETER componentPath
     Folder for the project to copy the project heads to.
-.PARAMETER heads
-    Which heads to include to copy, defaults to all. (Currently ignored.)
+
+.PARAMETER MultiTargets
+    Specifies the MultiTarget TFM(s) to include for building the components. The default value is 'all'.
```

Comment:
This is not the default, the default is (uwp, wasm, and wasdk)

Reply:
```suggestion
    Specifies the MultiTarget TFM(s) to include for building the components. The default value is 'uwp', 'wasm', 'wasdk'.
```